### ISSUE TITLE
[BITV] Files app is missing heading

### ISSUE THREAD

**Issue by ShGKme** [nextcloud/server#43619]
- Probably a regression from https://github.com/nextcloud/server/pull/43090
- In the past heading was set by https://github.com/nextcloud/server/blob/bb87232882f99c025ef224fb24aefd240b7d0c96/apps/files/src/views/Navigation.vue#L190
- This doesn't work on full Vue app. After migrating to `NcContent`, it has to be set here via `page-heading` prop: https://github.com/nextcloud/server/blob/bb87232882f99c025ef224fb24aefd240b7d0c96/apps/files/src/views/FilesList.vue#L23
